text/0135-compressed-blob-prefixes.md

```diff
@@ -0,0 +1,101 @@
+# RFC-0135: Compressed Blob Prefixes
+
+|                 |                                                                    |
+| --------------- | ------------------------------------------------------------------ |
+| **Start Date**  | 2025-01-06 |
+| **Description** | Standardize compressed blob prefixes |
+| **Authors**     | s0me0ne-unkn0wn (13WGadgNgqSjiGQvfhimw9pX26mvGdYQ6XgrjPANSEDRoGMt) |
+
+## Summary
+
+This RFC proposes a change that makes it possible to identify types of compressed blobs stored on-chain, as well as used off-chain, without the need for decompression.
+
+## Motivation
+
+Currently, a compressed blob does not give any idea of what's inside because the only thing that can be inside, according to the spec, is Wasm. In reality, other blob types are already being used, and more are to come. Blob decompression is a blocking operation, and routing the blob to a proper worker should not involve its decompression for the sake of efficiency. Thus, it is necessary to introduce a mechanism allowing to identify the blob type without decompressing it.
```

Comment:
While I think this RFC makes sense in general, I feel like we should be able to find better arguments than an implementation detail in the Parity implementation?

Reply:
Fair enough. I'll fix the wording.